README.md

```diff
@@ -76,8 +76,8 @@ module.exports = {
 
 - `@cybozu`
   - This is included in the all following presets
-- `@cybozu/eslint-config/presets/node`
-  - Including `eslint-plugin-node`
+- `@cybozu/eslint-config/presets/n`
```

Comment:
I think this name is still `@cybozu/eslint-config/presets/node`.
```suggestion
- `@cybozu/eslint-config/presets/node`
```

Reply:
Oh.. that's mistake.. 🙏 Thanks!